app/views/provider_interface/account_creation_in_progress.html.erb

```diff
@@ -6,13 +6,13 @@
       Account creation in progress
     </h1>
     <p class="govuk-body">
-    You are seeing this page because you have successfully signed in to the
-    Apply for teacher training service, but your user account has not yet been
-    associated with your training provider.
+      You are seeing this page because you have successfully signed in to the
+      <%= service_name %> service, but your user account has
```

Comment:
This previously said ‘Apply for teacher training’, but by referencing `service_name`, should now correctly say ‘Manage teacher training applications`.